test/extended/machines/cluster.go

```diff
@@ -78,7 +78,7 @@ var _ = g.Describe("[sig-node] Managed cluster", func() {
 		}
 	})
 
-	g.It("should report ready nodes the entire duration of the test run [Late]", func() {
+	g.It("should report ready nodes the entire duration of the test run [Late][apigroup:monitoring.openshift.io][apigroup:monitoring.coreos.com]", func() {
```

Comment:
@pacevedom would you please update the PR description to mention why both apigroups are needed?

Reply:
Done